src/components/Divider/Divider.js

```diff
@@ -0,0 +1,17 @@
+import React from "react"
+import s from "./DividerStyles.module.scss"
+import { InView } from "react-intersection-observer"
```

Comment:
Interesting that there's a package for IO. Do you understand this JS feature? Could you use it natively without a package? Is important to know these core JS things to fully understand them when using them :)